packages/lwc/scripts/utils/helpers.js

```diff
@@ -15,7 +15,7 @@ function createDir(dir) {
 }
 
 function getEs6ModuleEntry(pkg) {
-    const pkgFilePath = require.resolve(`${pkg}/package.json`);
+    const pkgFilePath = path.resolve(__dirname, '../../../', pkg, './package.json');
```

Comment:
I changed this because originally I was using package exports, which breaks if you try to require an unexposed file. I think it's better to just use `path.resolve` anyway though.